vector/src/main/java/im/vector/app/core/pushers/VectorUnifiedPushMessagingReceiver.kt

```diff
@@ -57,7 +58,10 @@ class VectorMessagingReceiver : MessagingReceiver() {
      * @param instance connection, for multi-account
      */
     override fun onMessage(context: Context, message: ByteArray, instance: String) {
-        vectorMessagingHelper.onMessage(String(message))
+        Timber.tag(loggerTag.value).d("New message")
+        pushParser.parsePushDataUnifiedPush(message)?.let {
+            vectorPushHandler.handle(it)
+        }
```

Comment:
Maybe keep the warning log, just in case it happens in the future:

```kotlin
?: run {
Timber.tag(loggerTag.value).w("Invalid received data Json format")
}
```